website/docs/cloud-docs/workspaces/settings/run-triggers.mdx

```diff
@@ -40,10 +43,12 @@ Under the "Source Workspaces" section, select the workspace you would like to co
 
 Runs which are queued in your workspace through a run trigger will include extra information in their run details section. This includes links to the source workspace and the successfully applied run that activated the run trigger.
 
+**SCREENSHOT FOR DELETION**
 ![Screenshot: a run's run details section](/img/docs/run-triggers-run-details.png)
 
 Operators of your source workspaces will be informed of the connection to your workspace during the [plan](/docs/glossary#plan-noun-1-) and [apply](/docs/glossary#apply-noun-) phases of their run.
```

Comment:
I was just thinking that maybe if we remove the screenshot we should make the text more specific here. Does this feel like it makes sense to you?

```suggestion
The source workspace includes a message in the [plan](/docs/glossary#plan-noun-1-) and [apply](/docs/glossary#apply-noun-) and apply run details that specifies the workspaces where Terraform Cloud will automatically start runs.